src/test/runTest.ts

```diff
@@ -11,9 +14,31 @@ async function main() {
         // The path to test runner
         // Passed to --extensionTestsPath
         const extensionTestsPath = path.resolve(__dirname, './unittest/index');
+        const vscodeExecutablePath = await downloadAndUnzipVSCode('stable');
+        const [cliPath, ...args] = resolveCliArgsFromVSCodeExecutablePath(vscodeExecutablePath);
 
-        // Download VS Code, unzip it and run the integration test
-        await runTests({ extensionDevelopmentPath, extensionTestsPath });
+        // Use cp.spawn / cp.exec for custom setup
+        if (getOSType() == 'Windows') {
```

Comment:
Avoid hardcoding strings where possible.
```suggestion
        if (getOSType() == OSType.Windows) {
```